rules/BUILD

```diff
@@ -13,6 +13,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
+
 # TODO(aiuto): Add dependency on Skylib for bzl_library so that we can later
 # generate docs with stardoc.
+
+load("@rules_license//rules:license.bzl", "license")
```

Comment:
The `license` rule is loaded but is unused, this line can be removed.
```suggestion
```

Reply:
Yes. And removed in the other places as well.